tests/vars/podman_tests.yml

```diff
@@ -0,0 +1,9 @@
+ara_tests_cleanup: true
+ara_api_root_dir: "{{ ansible_user_dir }}/.ara-tests"
+ara_api_secret_key: testing
+ara_api_install_method: 'podman'
+ara_api_version: 'latest'
```

Comment:
Ah, that's why it wasn't using master ;)

Reply:
Indeed ;)